docs/getting-started/quickstart.md

```diff
@@ -83,9 +83,9 @@ Once logged in, you're greeted with the option to create your first
 2. Click **"Create Collection"** and a side menu will appear.
 3. Fill in a **Name**.\
    _For the sake of this demo, we'll be calling ours `articles`, but feel free to make it your own!_
-4. Leave the other options at default. Click <span mi btn>arrow_forward</span> and the **"Optional System Fields"** menu
+4. Leave the other options at default. Click <span mi btn>arrow*forward</span> and the **"Optional System Fields"** menu
    will open.\
-   _Keep the values in this menu at the default, toggled off, for now. You can adjust them later._
+   \_Keep the values in this menu at the default, toggled off, for now. You can adjust them later.*
```

Comment:
@rijkvanzanten @jaycammarano 

Something weird is going on here, and i can't figure it out. Every time we make a commit, `arrow_forward` compiles to `arrow*forward`.... In the PR when I wrote this doc, I used `arrow\_forward` and that compiled into `arrow_forward` after the commit... But now, all commits appear to impact it.

Same weirdness on line 88, As you can see, it was written `_abc_` and then gets compiled to `_abc*` which is obviously subpar....

I spent at least 30 min trying to see if there was a `*` or `_` mismatch somewhere before and couldn't spot one.

Any ideas on what's up here?

Reply:
Thanks @azrikahar !
Based on that bug you sent, it looks like I probably have an extra `*` somewhere....
I will track it down.